com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -46,6 +48,8 @@ public enum RpcQueueProcessingTypes
         private bool m_ProcessUpdateStagesExternally;
         private bool m_IsNotUsingBatching;
 
+        internal NetworkManager m_NetworkManager;
```

Comment:
I just want to clarify, so apart from breaking the singleton dep, we are saying that RPCQueueContainers are things that should naturally be bound to one and only one NM, yes?  So each NM should have its own RPCQueueContainer?  I think this is true but just want to float it.

Reply:
Nope, you can now have as many as you like.  Its "network manager instance relationship" is based on how it is constructed.  The default is that the NetworkManager creates one RpcQueueContainer when it starts up.  However, if this needs to change and we need like 20, then we can do that where the NetworkManager instantiates the one instance.